README.md

```diff
@@ -107,3 +107,43 @@ export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
 export MM_ADMIN_TOKEN=j44acwd8obn78cdcx7koid4jkr
 make deploy
 ```
+
```

Comment:
nit: This isn't "useful" for most users landing on `README.md` who won't be triggering a release. Could we perhaps document this on https://developers.mattermost.com/ and link out instead?

Reply:
Thank you @lieut-data,

I personally prefer having all crucial project-related data concise in the README file. 

Moreover, the actual templating for new plugins includes release instructions. We iterate over those here:
https://github.com/mattermost/mattermost-plugin-starter-template/blob/master/README.md?plain=1#L113-L120

I don't have a strong opinion on this since it is mostly out of the scope of our work. Our main goal is to reach a consensus on pushing tags for release, thereby streamlining our process.

Let me know if you would like the documentation-related changes to be dropped.